BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
```

Comment:
In fact, every one of these steps should provide at least one link (the grunt-* task) to relevant documentation in addition to their explanations.